cmd/tikv-ctl/src/main.rs

```diff
@@ -654,30 +657,47 @@ fn run_sst_dump_command(args: Vec<String>, cfg: &TiKvConfig) {
     engine_rocks::raw::run_sst_dump_tool(&args, &opts);
 }
 
-fn print_bad_ssts(db: &str, manifest: Option<&str>, pd_client: RpcClient, cfg: &TiKvConfig) {
+fn print_bad_ssts(data_dir: &str, manifest: Option<&str>, pd_client: RpcClient, cfg: &TiKvConfig) {
+    let db = &cfg.infer_kv_engine_path(Some(data_dir)).unwrap();
+    println!(
```

Comment:
it's necessary?

Reply:
we updated some cmd args but not updated tikv-ctl doc, like `--db` is deprecated. This msg is usefull to help some users to know if they enter the correct path

Reply:
Okay. Which PR updated the args but not updated tikv-ctl doc, let's call the author to update tikv-ctl doc.

Reply:
FYI https://docs.pingcap.com/zh/tidb/dev/tikv-control don't give any explanation but exit when `[host|data-dir] is not specified`

Reply:
> Okay. Which PR updated the args but not updated tikv-ctl doc, let's call the author to update tikv-ctl doc.

cc @JayiceZ